static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -344,9 +349,7 @@ export function SourceMapsDebuggerModal({
               }/4)`}
               hidden={
                 !sourceResolutionResults.hasScrapingData ||
-                !sourceResolutionResults.sdkName?.startsWith(
-                  'sentry.javascript.react-native'
-                )
+                isReactNativeSDK({sdkName: sourceResolutionResults.sdkName})
```

Comment:
Hosting Publicly  shall not be shown for react-native and since this is minor, I am already fixing it here in this PR